src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -2021,6 +2021,25 @@ void Eltwise::initSupportedPrimitiveDescriptors() {
             IE_THROW() << "Eltwise node with name `" << getName() << "` doesn't support BF16 precision on this target.";
     }
 
+#if defined(OV_CPU_WITH_ACL)
+    Precision forcedPrec;
+    //ACL implementation supports only identical precisions on inputs/outputs so they are aligned it to highest one
+    if (AclEltwiseExecutor::isEltwiseAlgorithmSupported(getAlgorithm())) {
```

Comment:
As far as I understand most of the ACL algorithms doesn't support integer precisions (as well as ref impl only supports FP32). So to avoid pipeline failure I would propose to check if `forcedPrec` is integer (!is_float) and set in to FP32 in  that case.